control_panel_api/management/commands/load_kube_users.py

```diff
@@ -0,0 +1,42 @@
+import base64
+import logging
+
+from django.core.management.base import BaseCommand
+from django.db.utils import IntegrityError
+from kubernetes import config, client
+
+from control_panel_api.models import User
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Import Kubernetes user-secrets data into the User.  Uses KUBECONFIG envvar or defaults to ~/.kube/config"
+
+    USER_SECRETS_NAME = 'user-secrets'
+
+    def add_arguments(self, parser):
+        pass
+
+    def handle(self, *args, **options):
+        config.load_kube_config()
+        v1 = client.CoreV1Api()
+        results = v1.list_secret_for_all_namespaces()
+
+        user_secrets = [item.data for item in results.items if item.metadata.name == Command.USER_SECRETS_NAME]
```

Comment:
Yeah, don't worry, I was "thinking aloud" and filtering in memory is more than OK.